.pre-commit-config.yaml

```diff
@@ -32,6 +32,10 @@ repos:
                 additional_dependencies: [types-cachetools]
                 args: ["--module=dask_cuda", "--ignore-missing-imports"]
                 pass_filenames: false
+      - repo: https://github.com/rapidsai/pre-commit-hooks
+        rev: v0.3.0
```

Comment:
Most of RAPIDS is still using v0.2.0 IIRC. Is it important that we go through and update?

Reply:
v0.3.0 added a `--rapids-version` flag to the `verify-alpha-spec` hook, specifically to support `ucxx` and `ucx-py`. Most repos (including this one) don't need it, but I still use the newest version anyway.